In [28]:
import pandas as pd
df = pd.read_csv('/content/movie_review_data_1000.csv')
df.head()

review  sentiment
0  JESSICA: A GHOST STORY is as the name implies ...          0
1  After seeing Arthur on TV numerous times I lau...          1
2  Set in Venice mainly on the Lido, Visconti's "...          1
3  Morris and Reva Applebaum had been the toast o...          1
4  Based on fact, this is the story of a teenager...          1

In [0]:
review = list(df['review'].values)

In [30]:
from sklearn.feature_extraction.text import TfidfVectorizer
tfidf = TfidfVectorizer()
review_vect = tfidf.fit_transform(review)
print(review_vect.shape)

(1000, 18506)


####OK review vector have 18506 numeric vecotr for each 1000 reviws


In [0]:
df_sentiment = pd.DataFrame(review_vect.todense(), columns = ['col_'+ str(i+1) for i in range(review_vect.shape[1])])
df_sentiment['sentiment'] = df['sentiment']

In [34]:
df_sentiment.tail()

col_1  col_2  col_3  col_4  ...  col_18504  col_18505  col_18506  sentiment
995    0.0    0.0    0.0    0.0  ...        0.0        0.0        0.0          1
996    0.0    0.0    0.0    0.0  ...        0.0        0.0        0.0          1
997    0.0    0.0    0.0    0.0  ...        0.0        0.0        0.0          0
998    0.0    0.0    0.0    0.0  ...        0.0        0.0        0.0          1
999    0.0    0.0    0.0    0.0  ...        0.0        0.0        0.0          0

[5 rows x 18507 columns]

As we can see the output shows numeric vector represtation there are 5 row and 18506 columns for each review  and each word in the doc.